packages/manifold/contracts/edition/ManifoldERC721Edition.sol

```diff
@@ -47,83 +48,95 @@ contract ManifoldERC721Edition is CreatorExtension, ICreatorExtensionTokenURI, I
     /**
      * @dev See {IManifoldERC721Edition-totalSupply}.
      */
-    function totalSupply(address creator, uint256 series) external view override returns(uint256) {
-        return _totalSupply[creator][series];
+    function totalSupply(address creatorCore, uint256 instanceId) external view override returns(uint256) {
+        return _totalSupply[creatorCore][instanceId];
     }
 
-    /**
-     * @dev See {IManifoldERC721Edition-maxSupply}.
-     */
-    function maxSupply(address creator, uint256 series) external view override returns(uint256) {
-        return _maxSupply[creator][series];
-    }
 
     /**
-     * @dev See {IManifoldERC721Edition-createSeries}.
+     * @dev See {IManifoldERC721Edition-instanceExists}.
      */
-    function createSeries(address creator, uint256 maxSupply_, string calldata prefix) external override creatorAdminRequired(creator) returns(uint256) {
-        _currentSeries[creator] += 1;
-        uint256 series = _currentSeries[creator];
-        _maxSupply[creator][series] = maxSupply_;
-        _tokenPrefix[creator][series] = prefix;
-        emit SeriesCreated(msg.sender, creator, series, maxSupply_);
-        return series;
+    function instanceExists(address creatorCore, uint256 instanceId) external view override returns(bool) {
```

Comment:
Eh, could remove this if we have maxSupply function below.  And change maxSupply to revert if zero.  Probably cleaner and less code.

Reply:
done